StackTraceParser.csproj

```diff
@@ -1,64 +1,43 @@
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="nunit.framework, Version=2.6.4.14350, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
-      <HintPath>packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Public.cs" />
-    <Compile Include="StackTraceParser.cs" />
-    <Compile Include="StackTraceParserTests.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
+
   <ItemGroup>
-    <Folder Include="Properties\" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit" Version="3.12.0" PrivateAssets="All" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+    
   <PropertyGroup>
-    <PostBuildEvent>
-    </PostBuildEvent>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>StackTraceParser.Source</PackageId>
+    <Title>StackTraceParser (C# Source)</Title>
+    <PackageTags>stack trace parser diagnostics exception</PackageTags>
+    <Description>Parser for .NET and Mono stack traces.</Description>
+    <Copyright>Copyright &#169; 2020 Atif Aziz. All rights reserved.</Copyright>
+    <PackageVersion>1.3.1</PackageVersion>
+    <Authors>Atif Aziz</Authors>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <DefaultLanguage>en-us</DefaultLanguage>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageOutputPath>.\nugets</PackageOutputPath>
+    <PackageReleaseNotes>See https://github.com/atifaziz/StackTraceParser/milestone/3</PackageReleaseNotes>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+
+    <Target Name="AddPackageContent" BeforeTargets="GenerateNuspec">
+        <ItemGroup>
+            <_PackageFiles Include="StackTraceParser.cs;COPYING.txt">
+                <BuildAction>Compile</BuildAction>
+                <PackagePath>content\App_Packages\StackTraceParser.$(PackageVersion)\;contentFiles\cs\any\StackTraceParser\</PackagePath>
```

Comment:
What's the benefit you're after with adding the version in the path here?

Reply:
Non really other than seeing the version in the source folder. I'm not really opinionated about this. I'll change